bindata/glance/001-glance-secret.yaml

```diff
@@ -4,6 +4,6 @@ metadata:
   name: glance-secret
   namespace: {{ .Namespace }}
 stringData:
-  TransportUrl: rabbit://osp:passw0rd@rabbitmq.openstack.svc:5672/?ssl=0
+  TransportUrl: amqp://osp:passw0rd@amq-interconnect.openstack.svc:5672
```

Comment:
probably better if the controller just creates a transporturl secret and we pass the secret name into the CRs of the different components.